docroot/sites/classrooms.uiowa.edu/modules/classrooms_core/classrooms_core.module

```diff
@@ -31,6 +55,199 @@ function classrooms_core_form_alter(&$form, FormStateInterface $form_state, $for
       $form['field_room_scheduling_regions']['#disabled'] = TRUE;
       break;
 
+    case 'taxonomy_term_technology_features_form':
+
+      // Disable font awesome extra settings.
+      /** @var Drupal\uiowa_core\Access\UiowaCoreAccess $check */
+      $check = \Drupal::service('uiowa_core.access_checker');
+
+      /** @var Drupal\Core\Access\AccessResultInterface $access */
+      $access = $check->access(\Drupal::currentUser()->getAccount());
+
+      if ($access->isForbidden()) {
+        $form['field_technology_features_icon']['widget'][0]['settings']['size']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['fixed-width']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['border']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['invert']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['animation']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['additional_classes']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['duotone']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['masking']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['power_transforms']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['pull']['#disabled'] = TRUE;
+      }
+      break;
+
+  }
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_view().
+ */
+function classrooms_core_node_view(array &$build, EntityInterface $node, EntityViewDisplayInterface $display, $view_mode) {
+  if ($node->bundle() === "room" && $view_mode === "full") {
+    if ($display->getComponent('field_room_section_heading')) {
+      $build['field_room_section_heading'] = [
+        '#type' => 'markup',
+        '#markup' => '<h2 class="headline headline--serif headline--underline">' . t("Additional Information") . '</h2>',
+      ];
+    }
+    if (!$node->get('field_room_map')->isEmpty()) {
+      $room_url_map = $node->field_room_map
+        ->first()
+        ->get('entity')
+        ->getValue()
+        ->field_media_static_map
+        ->uri;
```

Comment:
We should probably use the `?->` to safely chain all these methods together without adding `if` checks.

Reply:
I added the null safe operator on these to chain them and safely access the properties and methods and avoid errors.